pkg/services/ngalert/remote/client/alertmanager.go

```diff
@@ -0,0 +1,111 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/grafana/grafana/pkg/infra/log"
+	amclient "github.com/prometheus/alertmanager/api/v2/client"
+)
+
+type AlertmanagerClient interface {
+	amclient.AlertmanagerAPI
+}
```

Comment:
Where would this interface be used?

Reply:
Good shout, I was playing around but couldn't figure out a way to make it work.